### PR TITLE
Category Section

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Added Single Category Page
2. Added API CALLS for the category

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -28,6 +28,12 @@ body {
...
 }
 
+.main-container {
+  grid-area: main;
+  background-color: var(--bg-container-color);
+  height: 100vh;
+}
+
 .main-grid-container {
   display: grid;
```
Comment (utkarshducoder) [inline src/App.css:38]: can use flex also

File: src/screens/home/category/Category.jsx [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
+import { useAxios } from "../../../hooks";
 
 export const Category = () => {
+  const [categories] = useAxios({
+    method: "get",
+    url: "/api/categories",
+  });
+
   return (
     <div className='category-wrapper'>
```
Comment (pankaj-prog) [inline src/screens/home/category/Category.jsx:16]: use semantic HTML, it is more comprehensible

**Review by dev-ashishsethi — APPROVED** [review]
everything looks good

**Review by pankaj-prog — APPROVED** [review]
The UI is impressive bro, and also the code is clean and readable. Awesome work

**Pull request merged**